diff -urN gnupg-1.0.5.org/g10/Makefile.am gnupg-1.0.5/g10/Makefile.am
--- gnupg-1.0.5.org/g10/Makefile.am	Mon Apr 30 17:44:20 2001
+++ gnupg-1.0.5/g10/Makefile.am	Mon Apr 30 17:51:44 2001
@@ -20,7 +20,6 @@
 
 INCLUDES = -I$(top_srcdir)/include -I$(top_srcdir)/intl
 EXTRA_DIST = OPTIONS  pubring.asc  options.skel
-OMIT_DEPENDENCIES = zlib.h zconf.h
 LDFLAGS = @LDFLAGS@ @DYNLINK_LDFLAGS@
 needed_libs = ../cipher/libcipher.a ../mpi/libmpi.a ../util/libutil.a
 
