--- gnupg-1.4.3/util/memory.c.orig	2005-07-27 19:39:24.000000000 +0200
+++ gnupg-1.4.3/util/memory.c	2006-04-23 22:33:50.770846250 +0200
@@ -509,15 +509,15 @@
         if( len >= n ) /* we don't shrink for now */
             return a;
         if( p[-1] == MAGIC_SEC_BYTE )
-            b = FNAME(alloc_secure_clear)(n FNAMEARG);
+            b = FNAMEXM(alloc_secure_clear)(n FNAMEARG);
         else
-            b = FNAME(alloc_clear)(n FNAMEARG);
+            b = FNAMEXM(alloc_clear)(n FNAMEARG);
         FNAME(check)(NULL FNAMEARG);
         memcpy(b, a, len );
-        FNAME(free)(p FNAMEARG);
+        FNAMEX(free)(p FNAMEARG);
     }
     else
-        b = FNAME(alloc)(n FNAMEARG);
+        b = FNAMEXM(alloc)(n FNAMEARG);
 #else
     if( m_is_secure(a) ) {
 	if( !(b = secmexrealloc( a, n )) )
