diff -urN gnupg-1.0.5.org/g10/g10.c gnupg-1.0.5/g10/g10.c
--- gnupg-1.0.5.org/g10/g10.c	Mon Apr 30 17:44:20 2001
+++ gnupg-1.0.5/g10/g10.c	Mon Apr 30 17:44:53 2001
@@ -513,12 +513,7 @@
     set_gettext_file( PACKAGE );
   #else
   #ifdef ENABLE_NLS
-    #ifdef HAVE_LC_MESSAGES
-       setlocale( LC_TIME, "" );
-       setlocale( LC_MESSAGES, "" );
-    #else
-       setlocale( LC_ALL, "" );
-    #endif
+    setlocale( LC_ALL, "" );
     bindtextdomain( PACKAGE, G10_LOCALEDIR );
     textdomain( PACKAGE );
   #endif
