--- gnupg-1.4.7/mpi/config.links.orig	2007-11-07 20:31:08.000000000 +0000
+++ gnupg-1.4.7/mpi/config.links	2007-11-07 20:36:30.000000000 +0000
@@ -125,11 +125,8 @@
 	mpi_extra_modules="udiv-qrnnd"
 	;;
     sparc64-*-linux-gnu)
-	# An extra rule because we have an report for this one only.
-	# Should be compared against the next GMP version
-	echo '/* configured for sparc64-*-linux-gnu */' >>./mpi/asm-syntax.h
-	path="sparc32v8 sparc32"
-	mpi_extra_modules="udiv"
+       echo '/* No working assembler modules available */' >>./mpi/asm-syntax.h
+       path=""
 	;;
     sparc64-sun-solaris2*)
 	# Got a report that udiv is missing, so we try this one
